resources/server/src/landingpage/osv-scanner.toml

```diff
@@ -0,0 +1,23 @@
+[[IgnoredVulns]]
+id = "GHSA-8jmw-wjr8-2x66"
+reason = "...the JS application is not built nor shipped anymore... Look to https://github.com/qgis/QGIS/pull/55748"
+
+[[IgnoredVulns]]
+id = "GHSA-pfrx-2q88-qq97"
+reason = "...the JS application is not built nor shipped anymore... Look to https://github.com/qgis/QGIS/pull/55748"
+
+[[IgnoredVulns]]
+id = "GHSA-rc47-6667-2j5j"
+reason = "...the JS application is not built nor shipped anymore... Look to https://github.com/qgis/QGIS/pull/55748"
+
+[[IgnoredVulns]]
+id = "GHSA-9c47-m6qq-7p4h"
+reason = "...the JS application is not built nor shipped anymore... Look to https://github.com/qgis/QGIS/pull/55748"
+
+[[IgnoredVulns]]
+id = "GHSA-7fh5-64p2-3v2j"
+reason = "...the JS application is not built nor shipped anymore... Look to https://github.com/qgis/QGIS/pull/55748"
```

Comment:
same as above

Reply:
Resolved in a98e897bdd5f32fad05eb81236d75a7a0beeeea8